.changeset/dull-spoons-clap.md

```diff
@@ -0,0 +1,7 @@
+---
+'@sap-ux-private/control-property-editor-common': minor
+'@sap-ux-private/preview-middleware-client': minor
+'@sap-ux/control-property-editor': minor
+---
```

Comment:
Changeset for `preview-middleware` is missing. Please add. Also for this change I believe that a `patch` update is sufficient.

Reply:
Added in: https://github.com/SAP/open-ux-tools/pull/1422/commits/9fccc9128640cb8f8dacebb1089b45fbdaef7768